v21.2/common-issues-to-monitor.md

```diff
@@ -0,0 +1,292 @@
+
+| Category | Recommendations                                                                                                                                                                                                                                                                                                                                                                                                                            |
+|----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Storage  | <ul><li>Provision volumes with {% include {{ page.version.version }}/prod-deployment/provision-storage.md %}.</li><li>{% include {{ page.version.version }}/prod-deployment/prod-guidance-store-volume.md %}</li><li>{% include {{ page.version.version }}/prod-deployment/prod-guidance-log-volume.md %}</li><li>See additional storage recommendations in the [Production Checklist](recommended-production-settings.html#storage).</li> |
+| Disk I/O | <ul><li>Disks must be able to achieve {% include {{ page.version.version }}/prod-deployment/provision-disk-io.md %}.</li><li>{% include {{ page.version.version }}/prod-deployment/prod-guidance-lvm.md %}</li><li>See additional disk I/O recommendations in the [Production Checklist](recommended-production-settings.html#disk-i-o).</li>                                                                                              |
+
+### Storage and disk monitoring
+
+Monitor storage capacity and disk performance:
+
+| Metric or event                                   | Description                                                                                                          |
+|---------------------------------------------------|----------------------------------------------------------------------------------------------------------------------|
+| [Storage capacity](#storage-capacity)             | The available and used disk capacity in the CockroachDB [store](cockroach-start.html#store).                         |
+| [Disk IOPS](#disk-iops)                           | The number of disk reads and writes in queue.                                                                        |
+| [Node heartbeat latency](#node-heartbeat-latency) | The time between [node liveness](cluster-setup-troubleshooting.html#node-liveness-issues) heartbeats.                |
+| [Command commit latency](#command-commit-latency) | The speed at which [Raft commands](architecture/replication-layer.html) are being committed by nodes in the cluster. |
+
+#### Storage capacity
```

Comment:
@a-entin, although it wasn't part of your Most Common Problems runbook, I also added this storage capacity section so that all the sizing guidance can be together. Let me know if we should make additional points here.